### PR TITLE
Add services section

### PR DESCRIPTION
**Pull request by MRDriks**

### CODE REVIEW

File: src/sass/style.scss [hunk 1]
```diff
@@ -9,6 +9,12 @@
     position: absolute;
 }
 
+%column-center-alignment {
```
Comment (yanosh-almashi) [inline src/sass/style.scss:12]: what's the use of % character here?

File: src/sass/style.scss [hunk 2]
```diff
@@ -101,15 +125,13 @@ p {
 }
 
 .header__title {
+    @extend %column-center-alignment;
+    justify-content: center;
```
Comment (yanosh-almashi) [inline src/sass/style.scss:129]: shouldn't this rule be added to %column-center-alignment class?

File: build/index.html [hunk 3]
```diff
@@ -46,8 +46,51 @@ <h2>A creative portfolio template</h2>
                 </div>
             </div>
         </section>
-        <section>
-            <div class="container"></div>
+        <section class="section-services">
```
Comment (yanosh-almashi) [inline build/index.html:49]: there is no need to name your section tag class as service-..., rename it to just "services"

File: build/index.html [hunk 4]
```diff
@@ -46,8 +46,51 @@ <h2>A creative portfolio template</h2>
...
+                            <hr>
+                            <p>
+                                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed a lorem quis 
+                                neque interdum consequat ut sed sem. Duis quis tempor nunc. Interdum et 
+                                malesuada fames ac ante ipsum primis in faucibus.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
```
Comment (yanosh-almashi) [inline build/index.html:93]: refactor classnames according to the comment above/

**Review by yanosh-almashi — CHANGES_REQUESTED** [review]

File: src/sass/style.scss (hunk 1 above)
Comment (MRDriks) [reply src/sass/style.scss:12]: In this block implemented template which allows inherit styles.

**Pull request closed without merging**